### DISCUSSION TITLE
resourcegroupstaggingapi get-resources issue with ApiGateway Apis

### DISCUSSION THREAD

**Discussion by klubi** [aws/aws-cli#8156] [Q&A]
Long story short:
> Does anyone know how to filter only apis using `--resource-type-filters` flag?

`aws resourcegroupstaggingapi get-resources --resource-type-filters apigateway:apis` returns APIS along with APIS and stages.

I'm trying to use [prometheus cloudwatch exporter](https://github.com/prometheus/cloudwatch_exporter/issues/574#issuecomment-1706420810), which uses that very API, and I'm struggling with filtering resources properly.

**Comment by tim-finnigan** [comment]
Hi @klubi thanks for reaching out. The CLI [get-resources](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/resourcegroupstaggingapi/get-resources.html) command corresponds to the [GetResources API](https://docs.aws.amazon.com/resourcegroupstagging/latest/APIReference/API_GetResources.html). I'm not familiar with https://github.com/prometheus/cloudwatch_exporter and not sure if I understand exactly what you're trying to do. Could you explain your use case a bit more? Which resources are you trying to filter?
